public/js/utils/redux/middleware/log.js

```diff
@@ -9,7 +9,7 @@
  */
 function log({ dispatch, getState }) {
   return next => action => {
-    console.log("[DISPATCH]", JSON.stringify(action, null, 2));
+    console.log(`[DISPATCH ${action.type}]`, action);
```

Comment:
> Ideally, this wouldn't be as much of a question because there would be a better redux action log :/

You can always install the redux devtools extension if you want a better log! :)